adafruit_matrixportal/network.py

```diff
@@ -295,15 +295,24 @@ def connect(self):
         self._wifi.neo_status(STATUS_CONNECTING)
         while not self._wifi.esp.is_connected:
             # secrets dictionary must contain 'ssid' and 'password' at a minimum
-            print("Connecting to AP", secrets["ssid"])
-            if secrets["ssid"] == "CHANGE ME" or secrets["password"] == "CHANGE ME":
-                change_me = "\n" + "*" * 45
-                change_me += "\nPlease update the 'secrets.py' file on your\n"
-                change_me += "CIRCUITPY drive to include your local WiFi\n"
-                change_me += "access point SSID name in 'ssid' and SSID\n"
-                change_me += "password in 'password'. Then save to reload!\n"
-                change_me += "*" * 45
-                raise OSError(change_me)
+            if "ssid" in secrets:
+                if isinstance(secrets["ssid"], (list, tuple)):
+                    print(
+                        'secrets["ssid"] is a list or a tuple.  I assume you know what you\'re doing'
+                    )
+                else:
+                    print("Connecting to AP", secrets["ssid"])
+                    if (
+                        secrets["ssid"] == "CHANGE ME"
+                        or secrets["password"] == "CHANGE ME"
+                    ):
+                        change_me = "\n" + "*" * 45
+                        change_me += "\nPlease update the 'secrets.py' file on your\n"
+                        change_me += "CIRCUITPY drive to include your local WiFi\n"
+                        change_me += "access point SSID name in 'ssid' and SSID\n"
+                        change_me += "password in 'password'. Then save to reload!\n"
+                        change_me += "*" * 45
+                        raise OSError(change_me)
```

Comment:
What is the purpose of the change for the code to check if a user has set their SSID/password yet?

Reply:
yes, accidental inclusion.  Sorry, I'll fix that.

Reply:
[this is part of another change I have on my side to allow a list of SSID and a list of passwords, allowing the device to connect to a different wifi, depending on which is available]